### DISCUSSION TITLE
Next JS Standalone build queries

### DISCUSSION THREAD

**Discussion by smo043** [vercel/next.js#33166] [Help] [answered]
Hello Team,

I am using the next js standalone experimental feature, which reduces the docker image size. I have a query: How do I add a few extra deps during prod build without declaring it in project `imports?`

**For Example:**

I have a `sharp` npm package in my package.json dependencies; since I am using the standalone build, how do I enforce the next js to include the sharp package `.next/standalone/node_modules` during build time?

Any help here would be highly appreciated.

**Comment by balazsorban44** [comment, marked as answer]
`sharp` is being traced as long as it is installed when you build the app. You might need to install `canary` for that, or wait until it is available on `latest`

**Reply by smo043** [reply]
1. Could you pls help when canary changes are planned for release? We are currently using 12.0.7, and it started failing in docker with the below error. I upgraded the version to `12.0.8-canary.20`, and the below error is fixed, but we don't want to go for production with canary versions.
If Team can help fix the below version and release a patch version, it will be very helpful.

```
Error: Cannot find module './image-optimizer'
Require stack:
- /app/node_modules/next/dist/server/next-server.js
- /app/server.js
```

2. Sharp is not being traced with `12.0.8-canary.20` during build time, it fails in docker. I have to install it manually inside .next/standalone/node_modules.

**Reply by balazsorban44** [reply]
1. Planning one very soon, keep an eye on the releases.
2. Could you create a minimal reproduction repository and open a bug report with that?

I previously moved this to discussions because the description was something else, it looks like you have heavily edited it since, and thus the issue seems to be something else now.

**Reply by smo043** [reply]
Sure I will create one example repo, and we can close this discussion.
